src/expr/proof_node_manager.cpp

```diff
@@ -22,7 +22,10 @@ using namespace CVC4::kind;
 
 namespace CVC4 {
 
-ProofNodeManager::ProofNodeManager(ProofChecker* pc) : d_checker(pc) {}
+ProofNodeManager::ProofNodeManager(ProofChecker* pc)
+    : d_checker(pc), d_true(NodeManager::currentNM()->mkConst<bool>(true))
```

Comment:
```suggestion
    : d_checker(pc), d_true(NodeManager::currentNM()->mkConst(true))
```

Reply:
Also I usually put this in the body of the constructor.

Reply:
Any reason to put it in the body?

Reply:
looks weird calling NodeManager::currentNM() in an initializer list.

Reply:
I see. I've changed it.